test/Unit/App/Middleware/ContentNegotiationMiddlewareTest.php

```diff
@@ -99,13 +99,6 @@ public function testCannotResolveRepresentation(): void
         );
     }
 
-    public function testFormatAcceptRequest(): void
```

Comment:
Instead of removing it, shouldn't we fix this test?
Like:
* calling `formatAcceptRequest('application/json')` and check if it returns the array `['json']`
* calling `formatAcceptRequest('application/json; text/xml')` and check if it returns the array `['json', 'xml']`
(not sure if I remember the behaviour correctly, but it should be something like the above)